packages/dialog/test/dialog.common.js

```diff
@@ -188,6 +188,29 @@ describe('vaadin-dialog', () => {
         expect(backdrop.hidden).to.be.true;
       });
     });
+
+    describe('closed event', () => {
+      it('should dispatch closed event when closed', async () => {
+        const closedSpy = sinon.spy();
+        listenOnce(dialog, 'closed', closedSpy);
```

Comment:
Could use `addEventListener` instead of `listenOnce`.  When using `listenOnce`, `expect(closedSpy.calledOnce).to.be.true;` wouldn't fail even if the event was dispatched twice